src/components/Tile/Tile.module.css

```
@@ -0,0 +1,101 @@
+  /* The engraved digit: a light top edge under dark ink reads as cut into the
+     ceramic rather than printed on it. */
+  text-shadow: 0 1px 0 rgb(255 255 255 / 85%);
+  box-shadow: var(--tile-edge);
+  /* Each state names how far it raises and tilts the tile instead of setting
+     transform itself, so a press adds its offset to whatever the state already
+     does. A press that set transform directly would replace the state's — a
+     marked tile would snap flat and un-rotate the moment it was touched. */
+  --tile-lift: 0px;
+  --tile-tilt: 0deg;
+  --tile-press: 0px;
+  transform: translateY(calc(var(--tile-press) - var(--tile-lift))) rotate(var(--tile-tilt));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)